src/components/PhoneNumberInput/index.js

```diff
@@ -26,27 +26,30 @@ window.contactoCountryData = Array.from(
 ).map((iso) => [regionNames.of(iso), iso, getCountryCallingCode(iso)])
 
 // Used to strip the numbers from the special characters
-// If the comparision is equal, we remove the last string - this helps in backspace functionality
-const stripSpecialChars = (a, b) => {
+const stripSpecialChars = (a) => {
   const regex = /[(-)\s+]/g
   const str = a?.replace(regex, '')
-  if (str?.length < 6 && str === b?.replace(regex, '')) return str?.substr(0, str.length - 1)
```

Comment:
This feature is not required as this is not specified in product, this was causing problem [point 7](https://plivo-team.atlassian.net/browse/CNTO-8564) of review